docs/source/concepts/cl_columns.rst

```diff
@@ -124,3 +124,93 @@ Contains a boolean value, with ``True`` indicating that the sample is likely to
 -----------
 Contains a score bounded between 0 and 1, which is used to determine whether a sample is an outlier. The lower the score of a sample, the more likely it is to be an outlier.
 
+
+Project Information Columns
```

Comment:
I don't think it makes sense to have the `Project Information Columns` be a subsection under the Cleanlab Columns page.

I think better to add a new page Project Information Columns, and link it directly under the "Concepts" page